deployment/entity-service/templates/api-deployment.yaml

```diff
@@ -94,6 +94,20 @@ spec:
             initialDelaySeconds: 60
             periodSeconds: 60
             timeoutSeconds: 5
+          {{- if .Values.workers.loggingCfg }}
```

Comment:
Suggest removing `workers` from the path as this logging configuration is for the whole service